### ISSUE TITLE
FAQ Followup - Add tutorial links and concise ramp information

### ISSUE THREAD

**Issue by kkukelka** [kodadot/nft-gallery#1962]
### **ON HOLD until tutorials are finished**
Followup issue regarding #1604

According to this comment

> I wrote to ramp network, to find out when buying KSM with fiat is going to be avalaible and they don't know. Question is, if it wouldn't be better to just not link it now and link it when it's avalaible again, or, when there's some functional dex on KSM, so people can exchange dot for ksm (dot is avalaible).
> As i tried to avoid it, link to ramp network is only in question number 6 for now.
> I'll also be making tutorials on most of these things, like how to create polkadot js wallet, how to simple mint, how to obtain KSM, how to sell your art, show several functions, what can one see in KodaDot and so on. So these questions will be updated and be ending with "if you have problems with this, you can watch this tutorial". But that will be once the tutorials are ready and proofed.
> 
> I guess we can then later continue in this issue and once the tutorial is ready on youtube i'll just send updated question number and we can update it. Also i would like to link directly to a playlist on KodaDot youtube channel with plenty of tutorials. Might as well do it all at once, the video creation shouldn't take long, as i'll be working on it from this point. 

extend FAQ question number 3 in the following manner
```
"3": "How can i mint NFTs?"
"In the left upper corner, you click on "create" and choose simple or classic mint. If you don't know how to continue, you can watch this tutorial."
```
1. and add tutorial interlinking

2. new information for ramps (maybe separate faq entry)

**Comment by cryptodamsky** [comment]
So, i can see that there are some mistakes in the FAQ(some hyperlinks wrong etc) so i'll update those

and i've finished one tutorial and that is - Polkadot.js tutorial. We'll see how the review goes on discord and will create Issue regarding this soon.

Once proofed, i'll send questions that need to be updated along with functional hyperlinks. The tutorial will be uploaded on KodaDot youtube so once this is done i'll be sending update. Sounds good? @kkukelka

**Comment by kkukelka** [comment]
> So, i can see that there are some mistakes in the FAQ(some hyperlinks wrong etc) so i'll update those
> 
> and i've finished one tutorial and that is - Polkadot.js tutorial. We'll see how the review goes on discord and will create Issue regarding this soon.
> 
> Once proofed, i'll send questions that need to be updated along with functional hyperlinks. The tutorial will be uploaded on KodaDot youtube so once this is done i'll be sending update. Sounds good? @kkukelka

Yes, sounds like a plan 👍 
Just comment on this issue here when everything is ready!

**Comment by cryptodamsky** [comment]
hey @kkukelka so i've been updating the faq basicly in our [docs ](https://docs.kodadot.xyz/FAQ.html#frequently-asked-questions)

What i mean now, what's the best flow on how to update current FAQ? I could see some links being wrongly linked in the current build, but i'm pretty sure all of them are working fine in the docs, because i updated them.

Can you advise on some ideal steps for both of us how to make this done?
Can surely send you here only questions that are new and also find the links which are wrongly linked, if that suits you. 

Or if it is easier for you to just copy the docs FAQ.md and put the same version to our FAQ on main page? 

Also grant links and credit function can be reverted as they are working flawlessly now.
I believe i'll be able to update some links with pull request later if i find some mistakes, since i kinda figured out the flow after my experience with docs :)

**Comment by kkukelka** [comment]
@cryptodamsky
I would suggest you only send me the new questions which should be added right here. also the number of the faq question so i know where to put it in terms of order. 
And of course the links that are broken in current build.

If you feel like something is missing afterwards and you feel confident in being able to do the required changes, you can of course make a PR in that regard. I can review it afterwards or answer you questions on Discord if you are unsure about stuff

**Comment by kkukelka** [comment]
@cryptodamsky all done. let me know if something is missing 👍

**Comment by cryptodamsky** [comment]
> @cryptodamsky PR is opened. let me know if something is missing 👍

Hey, also need to update discord link in question number "6" 

New link: https://discord.gg/657ReTFf78

**Comment by cryptodamsky** [comment]
Also question "12" needs to have both grant and discord links updated, can use the one i posted.

**Comment by kkukelka** [comment]
done
